src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActions/UnifiedFixAllSuggestedAction.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    /// <summary>
+    /// Similar to FixAllSuggestedAction, but in a location that can be used by
```

Comment:
Can we add an `IFixAllSuggestedAction` that is implemented by both `FixAllSuggestedAction` and `UnifiedFixAllSuggestedAction`?